src/server.js

```diff
@@ -43,7 +43,9 @@ import router from './router';
 import chunks from './chunk-manifest.json'; // eslint-disable-line import/no-unresolved
 import config from './config';
 
-require('dotenv').config();
+const dotenvConfig = require('dotenv').config();
+console.log('XXX JMF dotenvConfig', { dotenvConfig });
+console.log('XXX JMF process.env', {...process.env});
```

Comment:
This line isn't printing anything when I run `yarn start` locally...

>     XXX JMF process.env {}

Reply:
fixed in https://github.com/josephfrazier/reported-web/pull/427/commits/2837fe8baca5e32a419d6c0c00e41c2d7aa74981